### DISCUSSION TITLE
The /stonecutter command

### DISCUSSION THREAD

**Discussion by Sikatsu** [Zrips/CMI#2522] [Feature Requests]
**Description of feature/improvement request:**

We all know /enderchest and /workbench, some of my players requested to add /stonecutter for 1.14 servers.

**Comment by 456dev** [comment]
As far as my understanding of the spigot API goes, opening a crafting table is as simple as player.openworkbench(),
 where as opening other interactive guis, like furnaces, stonecutters and cartography tables require the use of NMS reflection code, increasing the complexity exponentially.

**Comment by Sikatsu** [comment]
Ahh I see. Well in that case i'd like a confirmation whether this will or will not be added in the future. So the issue can be closed. ^^

**Comment by Zrips** [comment]
A bigger issue is that some UI's require actual real blocks to be present, for example, the enchanting table needs to have bookshelves around it so it could calculate what level enchants it should give out.

**Comment by Sikatsu** [comment]
That's true, but for the stonecutter it's not required I believe?

**Comment by Zrips** [comment]
Maybe not, but i would need to dig around the base minecraft code to figure out the correct way in opening fake UI as spigot doesn't have one yet
